schema.sql

```diff
@@ -1,12 +1,27 @@
 /* Database schema to keep the structure of entire database. */
```

Comment:
I don´t understand this, I already have done this.

Reply:
Hi @AlucardSanin can you show me where you have done this? give me a screen shot please

Reply:
![image](https://user-images.githubusercontent.com/99287916/221243226-ded1e6be-4954-4f37-b7c9-ec06dabd7f8c.png)

Reply:
hi, @AlucardSanin I don't think this is the right way of doing this, you should use alter the query to update your table, it should look something like this   ALTER TABLE REMOVE species;
   ALTER TABLE animals ADD species_id int REFERENCES species(id);
   ALTER TABLE animals ADD owner_id int REFERENCES owners(id);

Reply:
hi @AlucardSanin I think from the start of Microverse, each project is separated from the past one, here we already have the animal and I think you already insert data into these tables, I don't think you should just change the code from the last project, you should make queries to change these tables, if we just want the tables to look like this we would ask you from the beginning.

Reply:
Usually from the start of Microverse, I think that we work on a feature in a branch, then we add new features and modifications to previous code when adding or using different approaches. Microverse kindly should specify in the project requirements `add queries` instead of `modify table` because that's actually what I did, I modified the table.

`Modify your inserted animals to include owner information (owner_id)`

Reply:
Hi @AlucardSanin , please remember that this project is not just seen by Microserse, all by code reviewers😉 in the feature an hr or a tech manager may see your code, and then to demonstrate your understanding you should right command to convince him that you understand this subject you will work in a large company in the feature and they won't let you just to change all there tables they may ask you to write some commands to update these tables.